Required for CMake 3.2 to work, precisely because FindOpenGL.cmake does not
check for X11 anymore.

commit 148ce85b26925e38ced2f196475086eb7884fb62
Author: Maxime Gauduin <alucryd@gmail.com>
Date:   Fri Mar 13 12:58:08 2015 +0100

    Include the FindX11 module
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -415,8 +445,7 @@
 			add_definitions(-DHAVE_WAYLAND=0)
 		endif(USE_WAYLAND AND WAYLAND_FOUND)
 
-		# Note: We do not need to explicitly check for X11 as it is done in the cmake
-		# FindOpenGL module on linux.
+               include(FindX11)
 		if(USE_X11 AND X11_FOUND)
 			set(USE_X11 1)
 			add_definitions(-DHAVE_X11=1)
